api/main_endpoints/routes/Messages.js

```diff
@@ -20,10 +20,28 @@ const numberOfConnections = {};
 const lastMessageSent = {};
 
 const writeMessage = ((roomId, message) => {
+
+  const currentTimestamp = new Date();
+
+  const messageObj = {
+    timestamp:
+      currentTimestamp.toLocaleDateString() + ' ' +
+      currentTimestamp.toLocaleTimeString(
+        [],
+        {
+          hour: '2-digit',
+          minute: '2-digit',
+          hour12: false
+        }
+      ),
```

Comment:
how about we return the epoch timetamp in ms, and the frontend does the parsing of it

```suggestion
    timestamp: Date.now(),
```